xcode/README.md

```diff
@@ -7,6 +7,8 @@ targets, two of which are the most likely to be of interest:
 
 * CapstoneStatic, producing `libcapstone.a`, Capstone as a static library;
 * CapstoneDynamic, producing `libcapstone.dylib`, Capstone as a shared library;
+* CapstoneOSXKernel, producing `libCapstoneOSXKernel.a`, Capstone as a static
```

Comment:
So that building CapstoneStatic does not overwrite the product of building CapstoneOSXKernel and vice versa.